modules/kubernetes/aks-core/variables.tf

```diff
@@ -647,3 +647,9 @@ variable "defender_enabled" {
   type        = bool
   default     = false
 }
+
+variable "use_coredns" {
```

Comment:
I suggest a different name for this. Maybe coredns_upstream or something similar instead.

Reply:
Indeed, that is better.